src/raftstore/errors.rs

```diff
@@ -25,10 +26,17 @@ use raft;
 
 use super::coprocessor::Error as CopError;
 use super::store::SnapError;
-use crate::util::{escape, transport};
+use crate::util::escape;
 
 pub const RAFTSTORE_IS_BUSY: &str = "raftstore is busy";
 
+#[derive(Debug, PartialEq, Clone, Copy)]
+pub enum DiscardReason {
+    Disconnected,
+    Filtered,
```

Comment:
Where do you use `Filtered`?

Reply:
In tests.

Reply:
What about `#[cfg(test)]`?

Reply:
No, integration tests links to the library with test profile disabled.

Reply:
Could you add some comments?